### ISSUE TITLE
Linux aarch64 support

### ISSUE THREAD

**Issue by corneliusroemer** [conda-forge/python-zlib-ng-feedstock#14]
### Comment:

Why not I guess :)

https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6143

**Comment by rhpvorderman** [comment]
Yes I want this too. Yes it should be supported (in fact the wheels are distributed). For some reason the aarch64 building did not trigger? I have no idea why this is the case. Thanks for looking into it!

I hope we can have full support on python-isal as well soon.

**Comment by hmaarrfk** [comment]
Seems like the 1.0.0 release has zlib-ng support.
https://pypi.org/project/zlib-ng/1.0.0/#files

Came to find somewhere to say cool work!
